src/components/TodoItem.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
Please try to fix linter errors instead of disabling linters.

Reply:
thank you for this but the issue is with the code provided from the tutorial 👍

Reply:
Considering that the tutorial provided code that caused those linter errors, you are right